composer.json

```diff
@@ -55,6 +55,7 @@
         "phpoffice/phpspreadsheet": "~1.5",
         "prestashop/blockwishlist": "^2.0",
         "prestashop/circuit-breaker": "^4.0",
+        "prestashop/classic": "dev-develop",
```

Comment:
classic-theme

Reply:
No, this here is the packagist name not the github name

https://packagist.org/packages/prestashop/classic

@eternoendless did on purpose set it as "classic" https://github.com/PrestaShop/classic-theme/commit/f339ecd5fe330bbc0593aab99d572f84922b2359
This is because it's redundant. We dont have psgdpr-module, we dont have ps_checkout-module : the suffix is ommitted

Reply:
The package name determines the folder where the package will be installed ([source](https://github.com/composer/installers/blob/main/src/Composer/Installers/PrestashopInstaller.php#L10)). The package must be named `classic`, or it would be put in `themes/classic-theme` and it wouldn't work.

Reply:
Is it planned to target a specific version later?

Reply:
Naturally yes, when we will have frozen development and released a version